graalvm/java11/cookiecutter-aws-sam-graalvm-maven/{{cookiecutter.project_name}}/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM amazonlinux:2
+
+RUN yum -y update \
+    && yum install -y unzip tar gzip bzip2-devel ed gcc gcc-c++ gcc-gfortran \
+    less libcurl-devel openssl openssl-devel readline-devel xz-devel \
+    zlib-devel glibc-static libcxx libcxx-devel llvm-toolset-7 zlib-static \
+    && rm -rf /var/cache/yum
+
+# Graal VM
+ENV JAVA_VERSION java11
+ENV GRAAL_VERSION 21.3.0
+ENV GRAAL_FOLDERNAME graalvm-ce-${JAVA_VERSION}-${GRAAL_VERSION}
+ENV GRAAL_FILENAME graalvm-ce-${JAVA_VERSION}-linux-amd64-${GRAAL_VERSION}.tar.gz
+RUN curl -4 -L https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-${GRAAL_VERSION}/${GRAAL_FILENAME} | tar -xvz
+RUN mv $GRAAL_FOLDERNAME /usr/lib/graalvm
+RUN rm -rf $GRAAL_FOLDERNAME
+
+# Maven
+ENV MVN_VERSION 3.8.4
+ENV MVN_FOLDERNAME apache-maven-${MVN_VERSION}
+ENV MVN_FILENAME apache-maven-${MVN_VERSION}-bin.tar.gz
+RUN curl -4 -L https://mirrors.ukfast.co.uk/sites/ftp.apache.org/maven/maven-3/${MVN_VERSION}/binaries/${MVN_FILENAME} | tar -xvz
```

Comment:
Can we use `https://dlcdn.apache.org/maven/maven-3/3.8.4/binaries/apache-maven-3.8.4-bin.tar.gz` instead of a mirror?